readme.md

```diff
@@ -99,13 +99,18 @@ View solution [here](http://codepen.io/awhitley1233/pen/EZyvMY)
 
 ![flexbox diagram](img/flexbox-diagram.jpg)
 
-When you declare `display: flex` on a container, it becomes a **flex container**.
+When you declare `display: flex` in a CSS rule, whatever is targeted by that rule becomes a **flex container**.
 
-Flexbox is distinctly different from the methods described above in that the arrangement of elements is managed by the **parent** container. The child of a **flex container** is called a **flex item**. By default, a flex item does not require any specific styling but item specific properties can be set which we will come to later.
+The flexbox approach differs from the methods described above in that the arrangement of elements is managed by the **parent** container.
+The child of a **flex container** is called a **flex-items**.
+We can change the way flex-items display by setting item-specific properties that will come later in the lesson.
```

Comment:
I would either put these all in the same paragraph or make sure there is an empty line between each section.